main.tf

```diff
@@ -69,60 +69,126 @@ resource "aws_route_table_association" "this" {
 }
 
 #creating security group
-resource "aws_security_group" "this" {
-  name        = "terraform_security_group"
-  description = "Allow 22, 80, 443 ports"
-  vpc_id      = aws_vpc.this.id
-#each ingress means one ingress rule
-  ingress {
-    description      = "HTTPS from EC2 instance"
-    from_port        = 443
-    to_port          = 443
-    protocol         = "tcp"
-    cidr_blocks      = ["0.0.0.0/0"]
-  }
+module "Security_group_EC2" {
+  source          = ".//modules/Security_group"
+  vpc_id          = "${aws_vpc.this.id}"
```

Comment:
Try to validate your code via terraform command. I guess it should throw some warning here :)

Reply:
![image](https://github.com/dliaudan/terraform-practice/assets/130764536/11b1bf06-3e70-48b2-9765-767496a74128)
Tried, there are no warnings, what here should be?

Reply:
<img width="589" alt="image" src="https://github.com/dliaudan/terraform-practice/assets/6039169/a04660da-f216-4f82-863e-90e0cefe501a">
You can pass value without using quotes ("")